torchbenchmark/models/fambench_xlmr/__init__.py

```diff
@@ -0,0 +1,83 @@
+import torch
+import sys
+import os
+from torchbenchmark import REPO_PATH
+from typing import Tuple
+
+# Import FAMBench model path
+class add_path():
+    def __init__(self, path):
+        self.path = path
+
+    def __enter__(self):
+        sys.path.insert(0, self.path)
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        try:
+            sys.path.remove(self.path)
+        except ValueError:
+            pass
+XLMR_PATH = os.path.join(REPO_PATH, "submodules", "FAMBench", "benchmarks", "xlmr", "ootb")
+with add_path(XLMR_PATH):
+    from xlmr import generate_dataset
+    from xlmr_parser import init_argparse
+
+from torchbenchmark.util.model import BenchmarkModel
+from torchbenchmark.tasks import NLP
+import torch.nn.functional as F
+
+class Model(BenchmarkModel):
+    task = NLP.LANGUAGE_MODELING
+    FAMBENCH_MODEL = True
+    # original parameters: FAMBench/benchmarks/run_xlmr_ootb.sh
```

Comment:
run_xlmr_ootb.sh is a dummy default config that is too small. Use this one. 

./run_xlmr_ootb.sh -c "--inference-only --famconfig=fb-1dev-A --num-batches=100 --batch-size=96 --sequence-length=64 --vocab-size=250000 --half-model --use-gpu --warmup-batches=20"